code/modules/power/lighting.dm

```diff
@@ -435,56 +436,86 @@
 	if(status == LIGHT_EMPTY)
 		to_chat(user, "There is no [fitting] in this light.")
 		return
+	if(user.a_intent == "help")
+		if(on)
+			to_chat(user, "This light [fitting] feels hot!")
+		else
+			to_chat(user, "The light [fitting] feels cold.")
+		return
+
+	if(world.time - last_clicked < 10) //makes the whole thing impossible to spam

```

Comment:
`changeNext_move(CLICK_CD_MELEE)`

Also, as @Crazylemon64 pointed out, he didn't suggest removing the rapid clicking, merely making it so that it required a different intent. to do so.

Ultimately, if someone wants to kill themselves on a bulb, let them---I know there's the whole brig argument, but those players will just resort to slapping or attacking themselves or shocking themselves instead---because that's just the kind of player they are.